packages/hockmd/hockmd.0.1.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+synopsis: "A library to access hackmd's api"
+description: "A library to access hackmd's api"
+maintainer: ["Paul-Elliot Anglès d'Auriac <peada@free.fr>"]
+authors: ["Paul-Elliot Anglès d'Auriac <peada@free.fr>"]
+license: "MIT"
+homepage: "https://github.com/panglesd/hockmd"
+doc: "https://panglesd.github.io/hockmd-api/"
```

Comment:
The link is wrong, it should be https://panglesd.github.io/hockmd/

Reply:
Thanks for spotting that! I opened #21722.